src/messages.ts

```diff
@@ -84,6 +86,10 @@ export function handleMessage(this: any, msg: { data: string }): void {
       return
     }
 
+    if (event.categoryCode === 'simulate') {
+      simulations$.error(event)
+    }
```

Comment:
```suggestion
    if (event.categoryCode === 'simulate') {
      simulations$.error(event)
      return
    }
```
Can we do an early return here since the event is handled?